DOCUMENTATION.md

```diff
@@ -185,13 +185,17 @@ email_reply_to_id='8e222534-7f05-4972-86e3-17c5d9f894e2' # optional UUID string
 
 If you omit this argument, the client will set your default email reply-to address for the notification.
 
-### Upload a document
+### Send a document by email
 
-You can upload a document to link to from an email notification.
+You can send files without the need for email attachments. First upload a document, then send recipients a link to download it.
 
```

Comment:
We could make it more positive - "You can send a document by sending an email with a link to that document".

Reply:
Agree that it could be more positive. However we felt there was a need to mention email attachments – and explain that this feature is different – because many users will expect attachments to be the solution to sending docs.

Also agree that we could be clearer about the fact that this adds a link to the doc to an email.